src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx

```diff
@@ -16,36 +16,86 @@ redirects:
 
-The more [conditions](/docs/alerts/new-relic-alerts-beta/configuring-alert-policies/define-alert-conditions) you define, the more [incidents](/docs/alerts/new-relic-alerts-beta/reviewing-alert-incidents/explore-incident-history) can be triggered and monitored. For example, your organization may need an alerting solution to accommodate extensive IT systems. Create policies with multiple conditions for [multiple monitored entities](/docs/alerts/new-relic-alerts-beta/configuring-alert-policies/select-product-targets-alert-condition) that notify you through one or more notification channels. Set [incident preference](/docs/alerts/new-relic-alerts/configuring-alert-policies/specify-when-new-relic-creates-incidents) to determine how violations lead to notifications.
+Do not rely on email for quick notifications in case of mail delays. An email list will also bounce if one is no longer valid.
 
-On the other hand, your organization may not need an extensive alerting structure. The fewer conditions you define and the more minimal your incident preference is, the fewer violations will be opened. For example, for a simple alerting solution, you could create a simple policy with only an email [notification channel](/docs/alerts/new-relic-alerts/managing-notification-channels/notification-channels-controlling-where-send-alerts).
 
-## Set thresholds for conditions [#conditions]
+## Incident preferences [#incident-practices]
 
-Set the [thresholds](/docs/alerts/new-relic-alerts-beta/configuring-alert-policies/define-thresholds-trigger-alert) for your policy's conditions to meaningful levels for your environment. Here are some suggested guidelines:
+Decide when you get incident notifications so you can respond to incidents when they happen.
+
+<Callout variant="tip">
+Learn more about your [incident preferences options](https://discuss.newrelic.com/t/relic-solution-alert-incident-preferences-are-the-key-to-consistent-alert-notifications/40867) if you’re new to Alerts.
+</Callout>
+
+The default incident preference setting combines all conditions within a policy into one incident. Change your default incident preference setting to better understand the exact scope and number of all violations.
+
+Each organization will have different needs, and each policy within an organization will have different needs too. Ask your team two important questions when deciding your incident preferences:
+- Do we want to be notified every time something goes wrong? 
+- Do we want to group all alike notifications together and be notified once?
+
+When a policy and its conditions have a broader scope (like managing several entities), increase your incident preference sensitivity. You will need more notifications because two incidents will not necessarily relate to each other.
```

Comment:
It's not so much "managing several entities" as "monitoring the performance of several entities", yeah?

Reply:
I think you need a little more on why you should increase your incident preference sensitivity. I mean, I think it's because you don't want to get too many notifications, right? 

The more things you're alerting on, the more notifications you can potentially get.